docs/DeveloperGuide.md

```diff
@@ -894,10 +895,10 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **Extensions**
 
-* 1a. The user input does not follow the format required.
+* 1a. The given arguments are invalid.
```

Comment:
The statement could be one or more of the given arguments are invalid.
The current statement could mean that only when all the given arguments are invalid then it gives error.

Reply:
I have changed the statement to "At least one of the given arguments is invalid.".